### DISCUSSION TITLE
Comments in wio-terminal code

### DISCUSSION THREAD

**Discussion by marty-optum** [microsoft/IoT-For-Beginners#328] [Q&A] [answered]
Do comments written in the source code (.cpp, .h, .ini) get uploaded to the Wio Terminal, or are they stripped out by the compiler before the upload?

I know that microcontrollers like the Wio Terminal have much less memory than General Purpose Controllers like the Raspberry Pi, so I need to know if I need to limit the amount of comments in the microcontroller code, or if it doesn't really matter.

Thank you.

**Comment by jimbobbennett** [comment, marked as answer]
They should be stripped out - the code is compiled, so things like comments that are not executable code won't make it into the final executable.

If you wanted to do a test, you could compile up a small binary, one with no comments, and one with a lot of comments, and you should see identical binary sizes.
